src/interdiff.c

```diff
@@ -58,6 +59,27 @@
 #define PATCH "patch"
 #endif
 
+#define COLOR_RESET     "\033[0m"
+
+/* Line type for coloring */
+enum line_type {
+	LINE_FILE = 0,
```

Comment:
The initial value for an enum without an explicit assignment is always 0. Remove ` = 0`.